src/com/couchbase/perf/sdk/stages/BuildDockerPythonSDKPerformer.groovy

```diff
@@ -33,10 +33,15 @@ class BuildDockerPythonSDKPerformer extends Stage{
         // Build context needs to be perf-sdk as we need the .proto files
         ctx.inSourceDirAbsolute {
             imp.dir('transactions-fit-performer') {
-                imp.dir('performers/python') {
-                    writeRequirementsFile(imp)
-                }
-                imp.execute("docker build -f performers/python/Dockerfile -t $imageName .")
+                for (int i = 0; i < 5; i ++) {
+                    try {
+                        imp.execute("docker build -f ./performers/python/Dockerfile -t $imageName .")
```

Comment:
Without the writeRequirementsFile stuff, this looks like it will always just build master.  But for performance testing we need to make sure we're building the performer against a specific version of the SDK.